.changeset/ten-penguins-roll.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-notifications': patch
```

Comment:
Now that we're making some breaking change this should be `minor`. I think immediate breakage is fine though